zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerCnxnFactory.java

```diff
@@ -40,6 +40,8 @@
 public abstract class ServerCnxnFactory {
 
     public static final String ZOOKEEPER_SERVER_CNXN_FACTORY = "zookeeper.serverCnxnFactory";
+    private static final String ZOOKEEPER_MAX_CONNECTION = "zookeeper.maxCnxns";
+    public static final int ZOOKEEPER_MAX_CONNECTION_DEFAULT = 0;
```

Comment:
`zookeeper.maxServerCnxns` is better? Corresponding to `maxClientCnxns`（no-binding）

Reply:
I was thinking "zookeeper.maxCnxns" would give a more wholesome meaning since we are looking at the maximum number of connections that the server can accept. Do u think we can keep the name as it is since it does not cause any confusion ?